web/modules/custom/dpl_fbi/src/GraphQL/Operations/WorkInfo/Work/Titles/WorkTitles.php

```diff
@@ -0,0 +1,46 @@
+<?php declare(strict_types=1);
```

Comment:
Why is this file (and all the others below) not being marked as code-generated here in Github?